dex/src/main/scala/com/wavesplatform/dex/db/LocalQueueStore.scala

```diff
@@ -1,38 +1,56 @@
 package com.wavesplatform.dex.db
 
+import cats.Monad
+import cats.implicits._
+
 import java.util.concurrent.ConcurrentLinkedQueue
 import java.util.concurrent.atomic.AtomicLong
-
 import com.google.common.primitives.{Longs, Shorts}
 import com.wavesplatform.dex.db.DbKeys._
-import com.wavesplatform.dex.db.leveldb.{DBExt, ReadOnlyDB}
+import com.wavesplatform.dex.db.leveldb.Key
 import com.wavesplatform.dex.queue.{ValidatedCommand, ValidatedCommandWithMeta}
-import org.iq80.leveldb.{DB, ReadOptions}
+import com.wavesplatform.dex.db.leveldb.LevelDb
+
+trait AbstractLocalQueueStore[F[_]] {
```

Comment:
`AssetPairsDb` has a small implementation comparing to `LocalQueueStore`. I think it would be more readable to define a separate class with implementation.
And do we need here in-mem implementation?

Reply:
Ok